docs/source/user/aerodyn-olaf/RunningOLAF.rst

```diff
@@ -17,80 +16,249 @@ available in the :ref:`installation` documentation.
+
+where the first approximation uses :math:`k_a a\approx 0.5` and the second approximation assumes a target distance of 6D.
+It is also possible to define the number of near wake panels based on a total number of revolution, :math:`n_\text{rot}`, leading to:
+
+.. math:: :label: ntargetRot
+   
+    n_\text{target, rotations} 
+       = \frac{n_\text{rot} T_\text{rot}}{dt_\text{FVW}}
+       = \frac{n_\text{rot} 2 \pi }{\Omega dt_\text{FVW}}
+       = \frac{n_\text{rot} 60 }{\Omega_\text{RPM} dt_\text{FVW}}
+       \qquad \text{(integer)}
+
+
+The wake of OLAF consists of two regions defined as "near wake" (NW) and "far wake" (FW), where the far consists of rolled-up tip and root vortices. 
+The far wake has reduced accuracy, therefore  velocity profiles crossing the far wake will likely be inaccurate. The role of the far wake is to reduce the computational time. For increases accuracy, it is recommended to use a longer near wake and a shorter far wake. The far-wake may be removed altogether.
+The far wake is further decomposed into two regions, a region where the vortex filaments are free to convect, and another one where the filaments convect with a fixed, averaged induced velocity. The advantage of having this "fixed" far wake region, is that it mitigates the impact of wake truncation  which is an erroneous boundary condition (vortex lines cannot end in the fluid). If the wake is truncated while still being "free", then the vorticity will rollup in this region. It is therefore useful for have this fixed far-wake region, the free far-wake region is not as important (in the future, OLAF will introduced a fixed near-wake region, so that the far-wake can be omitted altogether).
+The total number of wake panels is equal to the number of near-wake panels, free far-wake panels and fixed far-wake panels:
+
+.. math::
+   
+    n_\text{target} = n_\text{NW} + n_\text{FW} = n_\text{NW} + n_\text{FW,Free} + n_\text{FW,Fixed}
+
+OLAF input file defines
+:math:`n_\text{NW}`      (**nNWPanel**),
+:math:`n_\text{FW}`      (**nFWPanel**), and
+:math:`n_\text{FW,Free}` (**nFWPanelFree**).
+
+
+We currently recommend:
+
+- a total wake length of at least 6D (see Eq. :eq:`ntargetDist`), and
+- a total wake length corresponding to at least 10 rotor revolutions (see Eq. :eq:`ntargetRot`)
+- (depending on the operating conditions, one of the two conditions above will dominate, the largest wake length between the two is used).
+- a near-wake extent corresponding to at least 8 revolutions
+- a far-wake extent corresponding to 2 revolutions (including 1 revolution that is free)
+
+The python script provided at the end of this section implement these guidelines.
+
+General considerations:
+
+- If a far-wake is used, do not set it as "free" for more than half of the length (i.e. nFWPanelFree =< nFWPanel/2)
+- The near-wake is the most accurate. If computational time is not much of a concern, a long near-wake is preferred, with a short far-wake.
```

Comment:
I would only hyphenate "far-wake" if used to describe a noun and drop the hyphen otherwise.  I.e., "in the far-wake region" and "in the far wake".  Likewise below.